### PR TITLE
Added tests for fuzz

### PR DESCRIPTION
**Pull request by xarkes**
* Also removed unused functions

### CODE REVIEW

File: new/db/fuzz/fuzz [hunk 1]
```diff
@@ -0,0 +1,291 @@
+004d_000000006528c481_0000000000000000.inp
```
Comment (Maijin) [inline new/db/fuzz/fuzz]: This files is a bad idea as it requires manual work whereas current implementation just need a drag and drop, why can't we keep the old method ?

Comment (xarkes) [inline new/db/fuzz/fuzz]: cuz im a noob

Comment (Maijin) [inline new/db/fuzz/fuzz]: You're not a n00b, please fix so we keep the old way 👍

**Comment by xarkes** [comment]
Sounds better now, however there is an issue with the asynchronous tasks. The "Done" appears before the fuzzing part.

**Comment by xarkes** [comment]
Ok I also added the support for Windows platforms

**Comment by xarkes** [comment]
ping @radare

File: new/index.js [hunk 2]
```diff
@@ -38,10 +41,9 @@ class NewRegressions {
 
   callbackFromPath (from) {
     for (let row of [
-      ['db/cmd', this.runTest],
-      ['db/asm', this.runTestCmd],
-      ['db/dis', this.runTestDis],
-      ['db/bin', this.runTestBin]
+      [path.join('db', 'cmd'), this.runTest],
```
Comment (radare) [inline new/index.js]: Just hold 'cmd', 'asm'... and construct the full path with db later. Keep this list smaller

File: new/index.js [hunk 3]
```diff
@@ -62,32 +64,14 @@ class NewRegressions {
   runTestAsm (test, cb) {
     const self = this;
     return new Promise((resolve, reject) => {
-      // return resolve(cb(test));
```
Comment (radare) [inline new/index.js]: Ehy r u removing cmd and dis runtedt functions?

Comment (xarkes) [inline new/index.js]: dis is not useful anymore since the asm handles assembly and disassembly

**Pull request merged**